website/content/en/docs/upgrading/upgrade-guide.md

```diff
@@ -103,6 +103,7 @@ This version includes **dual support** for both alpha and beta APIs to ensure th
 Note that if you are rolling back after upgrading to `0.32.0`, note that `0.31.4` is the only version that supports handling rollback after you have deployed the v1beta1 APIs to your cluster.
 {{% /alert %}}
 
+* Karpenter now uses `settings.InterruptionQueue` instead of `settings.aws.InterruptionQueueName` in its helm chart. The CLI argument also changed to `--interruption-queue`.
```

Comment:
Looks like we just added this one section to the v0.32 section of the upgrade docs on the latest docs version. This change was covered in this section of the migration guide docs (https://karpenter.sh/v0.32/upgrading/v1beta1-migration/#global-settings). Given that this was the only "true" name change, I'm fine with including this in the upgrade note callouts but we should probably add this to every upgrade guide version so that it pops up in the v0.32 section on every version

Reply:
I was also thinking: does it make sense to generate a warning if a nodepool is configured with `capacity-type: spot` and no `interruptionQueue` set?

Reply:
added in https://github.com/aws/karpenter-provider-aws/pull/6062/commits/45e03b90f870f93f2235657eb02528effd562c3e, let me know what you think, I can revert if you don't like it 😄

Reply:
Made a comment on this below next to the change. Probably best to decouple this change from this docs update; I'm also not sure that it's a change that we would want to take. I think there are scenarios where you may not want to run the interruption queue and you are still comfortable using spot.